.cci.jenkinsfile

```diff
@@ -5,8 +5,8 @@
 COSA_IMAGE = 'quay.io/coreos-assembler/coreos-assembler:latest'
 COSA_BUILDROOT_IMAGE = 'registry.svc.ci.openshift.org/coreos/cosa-buildroot:latest'
 
-stage("Build") {
-parallel rpms: {
+stage("Build and sanitychecks") {
```

Comment:
Bikeshed/optional: maybe just `Build` ? And then have the parallel branches be e.g. `normal`, `minimal`, and `codestyle`?